bigtable/metrics_monitoring_exporter.go

```diff
@@ -86,31 +88,41 @@ func newMonitoringExporter(ctx context.Context, project string, opts ...option.C
 		client:    client,
 		shutdown:  make(chan struct{}),
 		projectID: project,
+		errLogger: errLogger,
 	}, nil
 }
 
+func (me *monitoringExporter) logErrorAndReturn(src string, err error) error {
```

Comment:
same as below, logging + returning error is a an anti pattern. If you are returning an error, it should be bubbled up and logged in a central location. Does the otel api not log this?

Reply:
Only wrapping now